@here/harp-mapview/lib/MapView.ts

```diff
@@ -1921,8 +1934,16 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Requests a redraw of the scene.
+     *
+     * @param forceRedraw If `true`, draws scene immediately, without waiting for the next
```

Comment:
Maybe add a clarifying comment that this also overrides the internal maxFps setting?